zoomx/src/main/AndroidManifest.xml

```diff
@@ -1 +1,11 @@
-<manifest package="com.zoomx.zoomx" />
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.zoomx.zoomx">
+
+    <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW" />
+
+    <application>
+        <activity android:name=".ui.requestlist.request.RequestActivity" />
+
+        <service android:name=".ui.requestlist.menu_view.ZoomxMenuService" />
```

Comment:
What do you think about renaming menu_view to menu as a package name? also try to put it under UI package directly.

Reply:
I will rename it. You renamed the package to view so I need first to merge with you then refactor it